_sass/_grid.scss

```diff
@@ -0,0 +1,92 @@
+$grids: (
+  ("1-1", (1fr 1fr), "-desktop"),
+  ("1-2", (1fr 2fr), "-desktop"),
+  ("2-1", (2fr 1fr), "-desktop"),
+  ("1-1-1", (1fr 1fr 1fr), "-desktop"),
+  ("1-1-1-1", (1fr 1fr 1fr 1fr), "-desktop"),
+  ("1-3", (1fr 3fr), ""),
+);
+
+$breakpoint-widths: (
+  "-desktop": $desktop-width,
+  "": 0
+);
+
+
+@each $grid in $grids {
+  // unpacking
+  $grid-name: nth($grid, 1);
+  $grid-ratio: nth($grid, 2);
+  $grid-prefix: nth($grid, 3);
+  $column-size: length($grid-ratio);
+  $breakpoint-width: map-get($breakpoint-widths, $grid-prefix);
+
+  @media (min-width: $breakpoint-width) {
+    .row#{$grid-prefix}-ratio-#{$grid-name} {
+      display: grid;
+      grid-template-columns: $grid-ratio;
+
+      .col-padded {
+        padding-left: 1em;
+        padding-right: 1em;
+      }
+
+      .col-padded:nth-child(#{$column-size}n - 1) {
```

Comment:
should probably be `#{$column-size}n`. items are indexed from 1 and the last cell in each row should simply be indexed at multiples of `$column-size`. e.g. on [this page](https://deploy-preview-268--acm-teach-la-website.netlify.app/citylab-cyber), passworks has both left and right padding, which might not be desirable

Reply:
Great catch! Resolved!